.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - "main"
+  pull_request: {}
+
+jobs:
+  checks:
+    name: Pre-merge Checks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Setup
+        run: npm ci
```

Comment:
You do need to explicitly `npm install`. Doesn't look like `npm ci` actually does anything.

Reply:
`npm ci` [should do the install](https://docs.npmjs.com/cli/v10/commands/npm-ci) (i think i hadn't properly formatted all these files yet though).

Reply:
Ah, interesting, as the previous CI failures were saying that `prettier` wasn't installed. But it's passing now!

Reply:
oooh, yeah that was a legit failure, i hadn't added prettier to the package.json yet 🤦